lib/smoney/company.rb

```diff
@@ -6,5 +6,6 @@ class Company < Entity::Base
 
     value :name
     value :siret
+    value :naf
```

Comment:
Unluckyly this is not the right field name. http://api.s-money.fr/documentation/utiliser-l-api/utilisateurs/